nifi-nar-bundles/nifi-smb-bundle/nifi-smb-processors/src/test/java/org/apache/nifi/processors/smb/ListSmbIT.java

```diff
@@ -202,11 +161,6 @@ public void shouldWriteFlowFileAttributesProperly() throws Exception {
                 .map(MockFlowFile::getAttributes)
                 .collect(toSet());
 
-        final Set<String> identifiers = allAttributes.stream()
-                .map(attributes -> attributes.get("identifier"))
-                .collect(toSet());
-        assertEquals(testFiles, identifiers);
-
         allAttributes.forEach(attribute -> assertEquals(
                 Stream.of(attribute.get("path"), attribute.get("filename")).filter(s -> !s.isEmpty()).collect(
                         Collectors.joining("/")),
```

Comment:
`absolute.path` attribute is no longer available. This test currently fails. Should remove this assertion.